project_euler/problem_58/sol1.py

```diff
@@ -25,21 +24,24 @@ def solution():
             # generating the value of n for 2n+1
             i += gap
             # generating the odd number
-            present_number = 2*i + 1
+            present_number = 2 * i + 1
             # appending the number to all_numbers
             all_numbers.append(present_number)
             # appending to prime if the number is prime
             if is_prime(present_number):
-                primes.append(2*i + 1)
+                primes.append(2 * i + 1)
         # changing the value of ratio at the end of loop
-        ratio = float(len(primes))/len(all_numbers)
+        ratio = float(len(primes)) / len(all_numbers)
         # increasing the gap after every 4 numbers
         gap += 1
+
     
-    return int((2*i+1)**0.5)
+    return int((2 * i + 1) ** 0.5)
 
 
 if __name__ == "__main__":
     from doctest import testmod
+
     testmod()
     print(solution())
+    
```

Comment:
I have added an extra line at end of the file still showing the same error.
How am I supposed to add it?